Superbuild/External_LIBTIFF.cmake

```diff
@@ -0,0 +1,54 @@
+############
+# This code was adapted from https://github.com/Slicer/Slicer/blob/main/SuperBuild/External_zlib.cmake
+############
+set(proj TIFF)
+
+set(${proj}_DEPENDENCIES "")
+
+if(Autoscoper_USE_SYSTEM_${proj})
+	unset(TIFF_ROOT CACHE)
+	find_package(TIFF REQUIRED)
+	set(TIFF_INCLUDE_DIR ${LIBTIFF_INCLUDE_DIR})
+	set(TIFF_LIBRARY ${LIBTIFF_LIBRARIES})
+endif()
+
+if(DEFINED TIFF_ROOT AND NOT EXISTS ${TIFF_ROOT})
+	message(FATAL_ERROR "TIFF_ROOT variable is defined but corresponds to nonexistent directory")
+endif()
+
+if(NOT DEFINED TIFF_ROOT AND NOT Autoscoper_USE_SYSTEM_${proj})
+	
+	set(EP_SOURCE_DIR ${CMAKE_BINARY_DIR}/${proj})
+	set(EP_BINARY_DIR ${CMAKE_BINARY_DIR}/${proj}-build)
+	set(EP_INSTALL_DIR ${CMAKE_INSTALL_PREFIX}/external)
+	
+	ExternalProject_Add(${proj}
+		GIT_REPOSITORY https://gitlab.com/libtiff/libtiff.git
+		GIT_TAG b6a17e567f143fab49734a9e09e5bafeb6f97354
+		SOURCE_DIR ${EP_SOURCE_DIR}
+		BINARY_DIR ${EP_BINARY_DIR}
+		INSTALL_DIR ${EP_INSTALL_DIR}
+		CMAKE_ARGS
+			-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
```

Comment:
Do not explicitly set `CMAKE_BUILD_TYPE`, building CMake dependency with the correct configuration is already handled by the CMake `ExternalPRoject` module.

See https://github.com/Kitware/CMake/blob/7b9757e50715ab6abc08e7005c66565f5c965de3/Modules/ExternalProject.cmake#L1811-L1861